agent/custom/bbc_connection_manager.py

```
@@ -563,5 +656,14 @@ def cleanup(self):
         mfaalog.info("[BbcConnectionManager] TCP连接已清理")
 
 
-# 全局实例
-bbc_manager = BbcConnectionManager()
+# 进程级单例（每个 agent 进程一个实例）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)